### PR TITLE
Install Make Light on 2.3.0

### PR DESCRIPTION
**Pull request by convolu**
towards https://github.com/KanoComputing/peldins/issues/2162

### CODE REVIEW

**Comment by convolu** [comment]
@tombettany @pazdera @radujipa Could you please review?

**Comment by radujipa** [comment]
:shipit:

File: kano_updater/scenarios.py [hunk 1]
```diff
@@ -361,6 +361,7 @@ def beta_210_to_beta_220(self):
 
     def beta_220_to_beta_230(self):
         install('rsync')
+        run_cmd_log('kano-apps install --no-gui powerup')
```
Comment (pazdera) [inline kano_updater/scenarios.py]: I think you need sudo here since the install is non-interactive. It won't be able to prompt you for a password.

Comment (convolu) [inline kano_updater/scenarios.py]: We've done in this way before https://github.com/KanoComputing/kano-updater/blob/jessie/kano_updater/scenarios.py#L287-L288 and https://github.com/KanoComputing/kano-updater/blob/jessie/kano_updater/scenarios.py#L327

I could change it if you want, just to be sure

Comment (pazdera) [inline kano_updater/scenarios.py]: @convolu No, it's correct. I started typing before thinking :-P

**Comment by convolu** [comment]
@tombettany @radujipa @pazdera I have reorganised the post update scenario. What do you guys think?

**Comment by radujipa** [comment]
The sign that a big one is coming :shipit:

**Comment by tombettany** [comment]
Looks good @convolu

**Comment by convolu** [comment]
I've updated as per your comments, @tombettany. I'll merge this to get time to test it.

**Pull request merged**